modules/default/calendar/calendar.js

```diff
@@ -710,16 +710,16 @@ Module.register("calendar", {
 		return string.charAt(0).toUpperCase() + string.slice(1);
 	},
 
-	/* titleTransform(title)
-	 * Transforms the title of an event for usage.
+	/* truncateTitle(title)
+	 * Transforms the title of an event and the location for usage.
 	 * Replaces parts of the text as defined in config.titleReplace.
 	 * Shortens title based on config.maxTitleLength and config.wrapEvents
 	 *
 	 * argument title string - The title to transform.
 	 *
 	 * return string - The transformed title.
 	 */
-	titleTransform: function (title) {
+	truncateTitle: function (title) {
 		for (var needle in this.config.titleReplace) {
 			var replacement = this.config.titleReplace[needle];
 
```

Comment:
Oh, this does more than just shorten the title, it also replaces stuff...  Maybe you only need line 735 ?

Reply:
Yes, this really transforms the title. The real question is the same here: should we do this transformation to the event title as well?
Anyway, I will revert these changes and introduce a new function which only really just strips the string.